tablo/api.py

```diff
@@ -480,6 +480,12 @@ def deploy(self, request, **kwargs):
                 dataset_id,
                 additional_fields=additional_fields
             )
+            add_geometry_column(dataset_id)
```

Comment:
It looks like the entire `try` block in which this occurs is identical to the other one below (lines 523-527). Can these be pulled out into a function that is callable from both places instead?

Reply:
Done.